### ISSUE TITLE
Space inserted between continuation records

### ISSUE THREAD

**Issue by orinmelio** [ministryofjustice/bai2#25]
Hi,
it seems like an unnecessary space is injected when concatenating continuation records (88).
should it be removed ?

https://github.com/ministryofjustice/bai2/blob/f8fd180503c3fe32252080f32b3d15a716dc4201/bai2/helpers.py#L14

**Comment by melvinpradep** [comment]
yes this can be removed, since its inserting additional space to continuation.

**Comment by JamesSQL** [comment]
You can remove the extra space but there is still one more problem. 

The code that generates the lines strips out all whitespace. If it does this, then any line that ends in a space that is supposed to be there will have the line removed. 

IE: (adding quotes for space visibility)
'ORIG BANK=JPMORGAN CHASE BANK, NA NEW YORK; CHECK '
'NO = 101'

Will get concatenated into ORIG BANK=JPMORGAN CHASE BANK, NA NEW YORK; CHECKNO = 101

If you change lines 40 and 41 on the bai_parser.py file to this (and remove the extra space in this file):
```
# only remove trailing linebreaks, never spaces
pr_line = line.rstrip('\r\n')
```

Then it will concatenate into this: ORIG BANK=JPMORGAN CHASE BANK, NA NEW YORK; CHECK NO = 101
And you won't have any extra or lost spaces. :)
